framework/ui/src/main/java/com/google/android/catalog/framework/ui/components/CatalogTopAppBar.kt

```diff
@@ -74,15 +91,24 @@ fun CatalogTopAppBar(
         },
         actions = {
             Box {
-                Row {
-                    if (selectedSample == null) {
+                Row(Modifier.animateContentSize()) {
+                    if (selectedSample == null || isDualPane) {
                         IconButton(onClick = onSearch) {
                             Icon(
                                 imageVector = Icons.Rounded.Search,
                                 contentDescription = "Search button"
                             )
                         }
                     }
+
+                    if (isDualPane && selectedSample != null) {
+                        IconButton(onClick = onExpand) {
+                            Icon(
+                                imageVector = Icons.Rounded.Fullscreen,
+                                contentDescription = "Search button"
```

Comment:
Nit: Should these content descriptions be in `strings.xml`?

Reply:
I will do an a11y pass in a next PR.